winterfell/controllers/authController.js

```diff
@@ -1,18 +1,17 @@
 var _ = require('lodash');
-var passport = require('passport');
 var constants = require('./../utils/constants');
 var http = require('http-status-codes');
 var httpErrors = require('./../utils/httpErrors');
+var Log = require('./../utils/logHelper');
+var passport = require('passport');
 var User = require('./../models/user');
 var UserValues = require('./../models/userValues');
 var UserService = require('./../services/userService');
 
 module.exports = function (app) {
 
   // Endpoint for routing sign-up
-  app.get('/signup', function(req, res) {
-    console.log('[signup] request received for ' + JSON.stringify(req.body));
-    console.log('[signup] session for ' + JSON.stringify(req.session));
+  app.get('/signup', [Log.api], function(req, res) {
```

Comment:
I guess I'm not totally sure what the concern about having static assets requests logged is. I would rather have all requests for that reason that if someone requests a static asset and gets a 500 or a 404, we are still gonna wanna know about it because its something to fix. We can maybe mess around with the log levels (do DEBUG for 20x/30x on static assets), but the logger middleware should be on every end point.